contents/blog/posthog-changelog.md

```diff
@@ -19,6 +19,33 @@ Interested about what new features, fixes and updates have just launched on Post
 
 > Some new features you read about below may still be in beta, behind a feature flag, or only available to paying users. Want to see which betas are currently available? [Check the roadmap](/roadmap)!
 
+## March 3, 2023
+
+### Hedgehog Mode
+![hog mode](../images/blog/array/hog_walk.gif)
+
+Let's get the biggest and most important updates out of the way first. We've updated our hedgehog mode, which you can access via the help menu in the app, to include some extra features. 
+
+You can now control Max ([that's his name](/handbook/company/team)) using the WASD keys, and we've also added a physics engine to give him a proper sense of gravity and inertia. You can make him jump with `W`, or spin with `S`. 
+
+### Dashboard templates
```

Comment:
added and just about to change 🤣 

@lharries maybe only need an update to the image?

Reply:
Here you go: https://github.com/PostHog/posthog/pull/14322

We can create a better image for this

Reply:
No need - I've subscribed to the PR, so I'll capture one when it's live. 👌